terraform/variables.tf

```diff
@@ -80,6 +80,16 @@ variable "acm_arn" {
   description = "The AWS ARN for the SSL certificate to be used. This is mandatory, but must be created before this step due to our hybrid DNS configuration"
 }
 
+variable "create_route53_entry" {
+  default     = false
+  description = "Should a route53 entry be created. If on-prem DNS is used, then don't create an entry in Route53."
+}
+
+variable "create_route53_zone" {
+  default     = false
+  description = "Should a route53 zone be created. If on-prem DNS is used, then create a zone in Route53."
```

Comment:
Should it be "If on-prem DNS is used, then _don't_ create a zone in Route53" instead?

Reply:
Oh yeah sorry that is super confusing. The zone can be created if we want to configure IPv4 and IPv6 alias records in route53 that our on-prem DNS can point to, however maybe I should remove it for now because I'm not even sure if we need it.